src/common/schemas/UserResponse.yml

```diff
@@ -2,12 +2,15 @@
     id:
       readOnly: true
       type: string
-    oauthID:
-      type: string
```

Comment:
`oauthID` isn't valid?

Reply:
`oAuthID` isn't returned by the response anymore in either cloud or oss